package.json

```diff
@@ -2,11 +2,15 @@
   "name": "room.inlive.app",
   "version": "0.1.0",
   "private": true,
+  "type": "commonjs",
   "scripts": {
     "create:env": "printenv > .env",
-    "dev": "next dev",
-    "build": "next build",
-    "start": "next start",
+    "dev:custom-server": "ts-node --project tsconfig.server.json ./app/server.ts",
+    "dev:next": "next dev",
+    "build:server": "tsc --project tsconfig.server.json",
+    "build:next": "next build",
+    "build": "npm run build:next && npm run build:server",
+    "start": "NODE_ENV=production node dist/server.js",
```

Comment:
Do we actually still need `dev:next` script? I think we don't need this script.
I think we can make the `dev` and `build` scripts simpler. Below scripts are much simpler. We don't need to update the readme too.

```
{
    "dev": "ts-node --project tsconfig.server.json ./app/server.ts",
    "build": "next build && tsc --project tsconfig.server.json",
}
```

Reply:
@GagahGK `dev:custom-server` should be changed to `dev` as default development script. Then, whenever we want to use built-in server, we can use the `dev:next`. For the `build` script, there is no need to separate it as `build:server` and `build:next` scripts

Reply:
then if that's the case then it's agreeable, but we might need to add a note about `dev` uses a custom server and its limitation about the hot reload ability

Reply:
Sure, better to give a little info about the custom server